server/models/Post.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const PostSchema = new Schema({
+    creator: Schema.Types.Mixed, // Mixed because it can be either user created
+    tags: [String],              // (ObjectId) or server created (String) from
+    title: {                     // fetching Discord, YouTube, Github).
```

Comment:
How should we account for posts we prepopulated? (e.g. the stuff we're fetching right now). I'm thinking about making the creator "server". This means the typed would have to be mixed.

Reply:
I think server is a good idea!! does it have to be mixed or can we just say string?

Reply:
Can we also add an id field for posts?

Reply:
> I think server is a good idea!! does it have to be mixed or can we just say string?

Oh thats a good point. We can probably just make the creator id the github username rather than the actual Mongo assigned ID


> Can we also add an id field for posts?

I think Mongo will automatically add a `_id` field? But i'd need to double check. 

While working on the meta issue for mocking up the frontend, I realized maybe we should also add a `type` field since the content will vary if its a youtube link vs text vs github issue/PR.

Reply:
Ooo yeah I think you're right about the mongo id field whoops haha 😅

Also yeah adding a type field would make things easier on the front end, agreed !!

Reply:
Awesome :) added the changes